autosklearn/evaluation/train_evaluator.py

```diff
@@ -571,7 +572,7 @@ def fit_predict_and_loss(self, iterative=False):
                 Y_test_pred = None
 
             self.Y_optimization = Y_targets
-            loss = self._loss(Y_targets, Y_optimization_pred)
+            self._loss(Y_targets, Y_optimization_pred)
```

Comment:
(again) I removed the variable assignment, but shouldn't we get rid of the line altogether?

Reply:
Yes, you can remove it if it's not used.

Reply:
removed